.circleci/config.yml

```diff
@@ -57,6 +57,22 @@ jobs:
           name: Ensure no changes to git-tracked files
           command: git --no-pager diff
 
+  pre-release-test:
+    machine:
+      image: ubuntu-2004:current
+    steps:
+      - checkout
+      - run: |
+          npm install -g fern-api@next
+          mkdir pre-release-test
+          cd pre-release-test
+          fern init --organization fern
+          fern add java
+          fern add typescript
+          fern add postman
+          fern generate
+          rm -rf pre-release-test
```

Comment:
i think there should be a `cd ..` first. and maybe `rm -r`, the `-f` is probably swallowing that error

Reply:
@dsinghvi

Reply:
ah okay